universum.cabal

```diff
@@ -1,6 +1,7 @@
 cabal-version:       2.2
 name:                universum
 version:             1.7.2
+x-revision:          1
```

Comment:
`x-revision` is said to be "merely an internal implementation detail of hackage": https://github.com/haskell/hackage-server/issues/779#issuecomment-405208631

So I am not sure if we should include it here :man_shrugging:

Reply:
Oh I see, I'll remove it then.